integration-tests/src/tests/features/in_memory_tries.rs

```diff
@@ -35,7 +35,9 @@ fn slow_test_in_memory_trie_node_consistency() {
     let mut clock = FakeClock::new(Utc::UNIX_EPOCH);
 
     let epoch_length = 9000;
-    let shard_layout = ShardLayout::simple_v1(&["account3", "account5", "account7"]);
+    let boundary_accounts =
+        ["account3", "account5", "account7"].iter().map(|a| a.parse().unwrap()).collect();
+    let shard_layout = ShardLayout::multi_shard_custom(boundary_accounts, 1);
```

Comment:
I see that every single call to this function is preceded with `.parse().unwrap()` stuff which IMO reduces the readability. I suggest changing  `multi_shard_custom` into something like this:
```rust
pub fn multi_shard_custom<T: AsRef<str>>(boundary_accounts: impl IntoIterator<Item = T>, version: ShardVersion) -> Self {

    let boundary_accounts: Vec<AccountId> =
            boundary_accounts.into_iter().map(|a| a.as_ref().parse().unwrap()).collect();
     // ...
}
```

Then it could be simply called like this:
```rust
ShardLayout::multi_shard_custom(["account3", "account5", "account7"], 1)
```

Reply:
I think I would still like to take the account_ids as input as in a lot of places we directly work with account_ids. It's not evident in this PR but we can take a look at the other use cases of `multi_shard_custom`

Reply:
The trick is that `AccountId` implements `AsRef<str>`, so those calls which use account IDs directly, can remain unchanged.